build_library/prod_image_util.sh

```diff
@@ -102,7 +103,13 @@ create_prod_image() {
   fi
 
   tar -cf "${BUILD_DIR}/${image_pkgdb}" -C "${root_fs_dir}" var/cache/edb var/db/pkg
+  sudo cp -r "${root_fs_dir}" "${BUILD_DIR}/root_fs_dir2" 
+  sudo rsync -a --delete  "${BUILD_DIR}/configroot/etc/portage" "${BUILD_DIR}/root_fs_dir2/etc" 
+  mksquashfs "${BUILD_DIR}/root_fs_dir2"  "${BUILD_DIR}/flatcar_production_image_sysext.squashfs" -noappend
 
```

Comment:
```suggestion
  sudo rm -rf "${BUILD_DIR}/root_fs_dir2"
```
We also need to clean it up